apps/api/src/app/events/events.controller.ts

```diff
@@ -22,12 +22,14 @@ import { TriggerEventToAll, TriggerEventToAllCommand } from './usecases/trigger-
 import { UserSession } from '../shared/framework/user.decorator';
 import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
 import { JwtAuthGuard } from '../auth/framework/auth.guard';
+import { LogDecorator } from '@novu/application-generic';
 
 @Controller({
   path: 'events',
   scope: Scope.REQUEST,
 })
 @ApiTags('Events')
+@LogDecorator()
```

Comment:
@Cliftonz not sure if you have answered this previously but couldn't find it, if nest-pino already have an auto logger for HTTP requests, why do we need the LogDecorator?